test/check-application

```diff
@@ -1597,6 +1597,19 @@ class TestFiles(testlib.MachineCase):
 
         self.enter_files()
 
+        def assert_bookmark(bookmark: str, *, exists: bool = True) -> None:
+            b.click("#bookmark-btn")
+            b.wait_visible(".pf-v5-c-menu")
+            if exists:
+                b.wait_visible(f".pf-v5-c-menu .pf-v5-c-menu__list-item:contains('{bookmark}') button")
+            else:
+                # Has to wait on file.watch() (inotify) to re-read the bookmarks
+                b.wait(lambda: bookmark not in b.text(".pf-v5-c-menu"))
+                self.assertNotIn(bookmark, b.text(".pf-v5-c-menu"))
```

Comment:
How is that different from `b.wait_not_in_text`?

Reply:
Not much! I'll change it to our helper